platforms/quorum/charts/node_constellation/templates/deployment.yaml

```diff
@@ -189,8 +189,15 @@ spec:
           echo slept for 5 to give transaction manager time to start;
 
           args=""
+          NODE_STATUS=""
           if [ $CONSENSUS == 'raft' ]; then
-            args=" --raft --raftport {{ .Values.node.ports.raft }} --rpcapi admin,db,eth,debug,miner,net,shh,txpool,personal,web3,quorum,raft "
+            NODE_STATUS={{ .Values.node.status }}
```

Comment:
Ensure that this is removed when ibft code is implemented. It should rather be if {{ .Values.node.status}} is defined then NODE_STATUS={{ .Values.node.status}}

Reply:
This will work with IBFT too as above line 193, I am setting this var to an empty string.
Earlier I was adding it in env vars, that would have failed if raft was not used. But this will do fine.